### PR TITLE
allow easy setting of host and port

### PR DESCRIPTION
**Pull request by marcoow**
I'm aware #8 had been closed before but I'm not really sure what the downsides of making setting the host and port easy are. How things currently are, I'd have to subclass `ExpressHTTPServer` only to change the host and it doesn't really seem clear whether that should be used by anyone anyway (see discussion here: https://github.com/ember-fastboot/fastboot-app-server/pull/34#discussion_r91873431)

I think providing options for both `host` and `port` is even better than allowing env vars as its more flexible.

### CODE REVIEW

**Comment by marcoow** [comment]
Currently setting the host isn't possible at all. On our server that results in the server only listening on the IPv6 interface which is a problem as we use IPv4 for the requests.

In general, I think it's more flexible to accept `host` and `port` as options anyway as you might not always be able to set or control the names of env vars or might get your config from elsewhere, e.g. files etc.

**Comment by marcoow** [comment]
@kratiahuja: updated the README

File: README.md [hunk 1]
```diff
@@ -54,6 +54,18 @@ $ PORT=8000 node server.js
 This will start an HTTP server on port 8000. To stop the server, type
 `Ctrl-C`.
 
+The host and port the server listens on can also be set via options when
+creating the server instance:
+
+```js
+let server = new FastBootAppServer({
+  distPath: '0.0.0.0',
```
Comment (krasnoukhov) [inline README.md]: Looks like a typo 👀

**Comment by krasnoukhov** [comment]
This is very useful change 👍 . People who use reverse proxy don't usually want to bind to `0.0.0.0` which is current default

File: README.md (hunk 1 above)
Comment (marcoow) [inline README.md]: 🤦‍♂️

Comment (marcoow) [inline README.md]: fixed

**Comment by marcoow** [comment]
@kratiahuja not sure I understand - I updated the README in 10c1334913fb5f42b71229f084720cf9a431b284.

File: README.md [hunk 2]
```diff
@@ -54,6 +54,18 @@ $ PORT=8000 node server.js
 This will start an HTTP server on port 8000. To stop the server, type
 `Ctrl-C`.
 
+The host and port the server listens on can also be set via options when
+creating the server instance:
+
+```js
+let server = new FastBootAppServer({
```
Comment (kratiahuja) [inline README.md]: Can this example be merged with the other optional options like (gzip) above?

**Comment by marcoow** [comment]
@kratiahuja done

**Comment by kratiahuja** [comment]
@marcoow Thanks for working on this. I'll release in a few hours.

**Comment by kratiahuja** [comment]
@marcoow fastboot-app-server@1.0.1 is published!

**Pull request merged**